extensions/amp-analytics/0.1/iframe-transport.js

```diff
@@ -24,6 +24,12 @@ import {setStyles} from '../../../src/style';
 import {hasOwn} from '../../../src/utils/object';
 import {IframeTransportMessageQueue} from './iframe-transport-message-queue';
 
+/** @private @const {string} */
+const TAG_ = 'amp-analytics.IframeTransport';
+
+/** @private @const {number} */
+const LONG_TASK_REPORTING_THRESHOLD_ = 5;
```

Comment:
Long task seems to fire about 3 times during iframe startup, so I completely arbitrarily set this to (that + 2). Open to other opinions.